lib/ui/painting.dart

```diff
@@ -1025,7 +1025,7 @@ enum Clip {
 // in the given dimension.
 //
 // This needs to be kept in sync with "kDoNotResizeDimension" in codec.cc
-const int _kDoNotResizeDimension = -1;
+const int kDoNotResizeDimension = -1;
```

Comment:
nit: If this is public, it should also have public DartDoc.

https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#documentation-dartdocs-javadocs-etc